File: src/main/java/com/thirdwavelist/barista/service/CafeService.java
package com.thirdwavelist.barista.service;

import com.thirdwavelist.barista.entity.Cafe;
import com.thirdwavelist.barista.repository.CafeRepository;
import org.springframework.stereotype.Service;

import java.util.Collection;

@Service
public class CafeService {
    private final CafeRepository cafeRepository;

    public CafeService(CafeRepository cafeRepository) {
        this.cafeRepository = cafeRepository;
    }

    public Collection<Cafe> getAll() {
        return cafeRepository.getAll();
    }

    public Collection<Cafe> getCafe(String id) {
        return cafeRepository.getById(id);
    }
}

File: src/main/java/com/thirdwavelist/barista/service/CafeService.kt
package com.thirdwavelist.barista.service

import com.thirdwavelist.barista.entity.Cafe
import com.thirdwavelist.barista.repository.CafeRepository
import org.springframework.stereotype.Service

@Service
class CafeService(private val cafeRepository: CafeRepository) {
    fun getAll(): Collection<Cafe> {
        return cafeRepository.getAll()
    }

    fun getById(id: String): Collection<Cafe> {
        return cafeRepository.getById(id)
    }
}